_episodes/08-crop-raster-data.md

```diff
@@ -126,41 +127,21 @@ Datum: Amersfoort
 ~~~
 {: .output}
 
-As seen, the coordinate systems differ. To crop the raster using the shapefile,
-we first convert the coordinate system of `cf_boundary_crop` to the coordinate
-system of `true_color_image`, and then check the coverage:
+As seen, the coordinate systems differ. To crop the raster using the shapefile, we first need to uniform the CRS. We can convert one to the other. But considering the size of `raster` is huge, to avoid loading the entire image, we will convert the coordinate system of `fields` to the coordinate system of `raster`:
```

Comment:
```suggestion
As seen, the coordinate systems differ. To crop the raster using the shapefile, we first need to reproject one dataset to the other's CRS. Since `raster` is large, we will convert the CRS of `fields` to the CRS of `raster` to avoid loading the entire image:
```